site/src/api/api.ts

```diff
@@ -1555,17 +1555,9 @@ export const getInsightsTemplate = async (
   return response.data;
 };
 
-export interface Health {
-  healthy: boolean;
-  time: string;
-  coder_version: string;
-  access_url: { healthy: boolean };
-  database: { healthy: boolean };
-  derp: { healthy: boolean };
-  websocket: { healthy: boolean };
-}
-
 export const getHealth = async () => {
-  const response = await axios.get<Health>("/api/v2/debug/health");
+  const response = await axios.get<TypesGen.HealthcheckReport>(
```

Comment:
@BrunoQuaresma It seems that `make gen` does not generate `DerphealthcheckReport` even though it is marked to be generated. Do you have any idea why it happens? 

```
@@ -2097,7 +2097,9 @@ export interface HealthcheckReport {
   readonly healthy: boolean;
   readonly warning: boolean;
   readonly failing_sections: string[];
-  readonly derp: DerphealthReport;
+  // Named type "github.com/coder/coder/v2/coderd/healthcheck/derphealth.Report" unknown, using "any"
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any -- External type
+  readonly derp: any;
   readonly access_url: HealthcheckAccessURLReport;
   readonly websocket: HealthcheckWebsocketReport;
   readonly database: HealthcheckDatabaseReport
```

Reply:
Extracted it here: https://github.com/coder/coder/pull/10650